.github/workflows/continuous-integration.yml

```diff
@@ -7,110 +7,95 @@ on:
 
 
 jobs:
-  eslint-front:
-    name: "ESLint: front"
+  eslint:
+    name: "ESLint"
     runs-on: ubuntu-latest
-    defaults:
-      run:
-        working-directory: ./front
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
       - name: Setup node
         uses: actions/setup-node@v4
         with:
-          node-version-file: 'front/.nvmrc'
+          node-version-file: '.nvmrc'
           cache: yarn
-          cache-dependency-path: "front/yarn.lock"
+          cache-dependency-path: "yarn.lock"
       - name: Install dependencies
         run: yarn install --immutable
       - name: Run ESLint
         run: yarn lint
 
-  prettier-front:
-    name: "Prettier: front"
+  prettier:
```

Comment:
hmm ok, why not keep separate actions, just like you did with jest that runs only for the front ?